### PR TITLE
Code style refactoring

### PR DESCRIPTION
**Pull request by ArthurDorokin**
refactoring complete

### CODE REVIEW

File: my-app/src/components/mainBanner/mainBanner.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './mainBanner.css';
 
-// make shorter
-const MainBanner = () => {
-    return (
+
+const MainBanner = () =>
```
Comment (megabatz) [inline my-app/src/components/mainBanner/mainBanner.jsx]: must be const MainBanner = () => (

File: my-app/src/components/content/content.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import './content.css';
-import data from '../constants.js';
+import {data} from '../constants.js';
```
Comment (megabatz) [inline my-app/src/components/content/content.jsx]: make more informative constant name

File: my-app/src/components/mainBanner/mainBanner.jsx (hunk 1 above)
Comment (ArthurDorokin) [reply my-app/src/components/mainBanner/mainBanner.jsx]: +

File: my-app/src/components/content/content.jsx (hunk 2 above)
Comment (ArthurDorokin) [reply my-app/src/components/content/content.jsx]: +

File: my-app/src/components/footer/footer.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './footer.css';
+import footerList from '../constants.js';
+
+const Footer = () => {
```
Comment (megabatz) [inline my-app/src/components/footer/footer.jsx]: can be shorter

**Comment by ArthurDorokin** [comment]
ok

**Pull request merged**